indico/modules/events/settings.py

```diff
@@ -261,3 +262,9 @@ def get_themes_for(self, event_type):
     'emails': [],
     'phones': []
 })
+
+event_contribution_settings = EventSettingsProxy('contrib', {
+    'contrib_duration': timedelta(minutes=20)
```

Comment:
'default_duration' is probably better now that it's in a contribution-specific settings proxy